build_metal.sh

```diff
@@ -27,6 +32,8 @@ show_help() {
     echo "  --clean                          Remove build workspaces."
     echo "  --build-static-libs              Build tt_metal (not ttnn) as a static lib (BUILD_SHARED_LIBS=OFF)"
     echo "  --disable-unity-builds           Disable Unity builds"
+    echo "  --cxx-compiler-path              Set path to C++ compiler."
+    echo "  --c-compiler-path                Set path to C++ compiler."
```

Comment:
This will cause merge conflict, the upstream already has this change.

Reply:
I'm not sure I understand why this presents a block. Git and github facilitate resolving these types of  conflicts.

Reply:
if two commits both modified the same lines, there will be merge conflict.

If they aren't on the same lines, then we could end up in a situation where we have these flags doubled...

https://github.com/tenstorrent/tt-metal/blob/2e970746bbe8cbeb47e00e43c2c352f2a783151a/build_metal.sh#L32-L33